dotnet/samples/KernelSyntaxExamples/Example40_DIContainer.cs

```diff
@@ -72,7 +72,9 @@ private static async Task UseKernelInDIPowerApp_AdvancedScenarioAsync()
 
         //Registering AI services Kernel is going to use
         var aiServicesCollection = new AIServiceCollection();
-        aiServicesCollection.SetService<ITextCompletion>(() => new OpenAITextCompletion("text-davinci-002", TestConfiguration.OpenAI.ApiKey));
+        aiServicesCollection.SetService<IChatCompletion>(() => new AzureChatCompletion("gpt-35-turbo",
```

Comment:
Replace "gpt-35-turbo" with configuration value?

Reply:
Yes.  Looks like I forgot a few. I just did a grep for "turbo" and found others. Changing them and sending a new iteration out.